### ISSUE TITLE
I'm curious about something. May I ask where all these map tiles come from?

### ISSUE THREAD

**Issue by hankexu** [Slluxx/TOTK-Interactive-Map#13]

**Comment by Slluxx** [comment]
Out of the games romfs section. I had to extract them, merge them painfully and then resplit them into a format that can be understood by leaflet. They are hosted on a different branch on this repository.

**Comment by takiyao** [comment]
> Out of the games romfs section. I had to extract them, merge them painfully and then resplit them into a format that can be understood by leaflet. They are hosted on a different branch on this repository.

Do you by any chance still have merged image of the map? I want to redraw the map, but unfortunately none of the versions found online are big enough to appreciate the details (save the interactive versions, which sadly I cant import to photoshop).

**Comment by Slluxx** [comment]
No, sorry. I do not have the original anymore. What's the issue with taking my map and just combining it again? It's all in the respective branch in this project.

**Comment by takiyao** [comment]
Simply put, I lack the knowledge you have to understand the logic behind the slice placements in the different folders in order to piece them together. I will figure something out eventually. Thank you for your time.

**Comment by Slluxx** [comment]
> Simply put, I lack the knowledge you have to understand the logic behind the slice placements in the different folders in order to piece them together. I will figure something out eventually. Thank you for your time.

Common, its not that hard to look at the website and see which image is loaded and where its placed.

![image](https://github.com/Slluxx/TOTK-Interactive-Map/assets/15162905/e3a423da-376e-4f86-a106-350235309d96)
![image](https://github.com/Slluxx/TOTK-Interactive-Map/assets/15162905/0462d584-9312-419e-9b59-07fa3ba4ca5e)

Based on that, you know everything that you need to know. 
`groundtiles/z/x/y.png`
z = zoom (8 is most detailed / zoomed in)
x = left to right
y = top to bottom
